microhapulator/data/template.html

```diff
@@ -319,11 +322,10 @@ <h2>Genotype Calling</h2>
                 The haplotype tallies, after all filters have been applied, represents the <em>genotype call</em> for that sample.
             </p>
             <p>
-                For this run, MicroHapulator applied a global detection threshold of <strong>&#8805;{{static}} reads</strong> and a global analytical threshold of <strong>&#8805;{{"{:.1f}".format(dynamic*100)}}% of total reads</strong>.
-                {% if filter_config is not none %}
-                User-provided per-marker thresholds were also applied as shown below.
+                Thresholds were were applied to each marker as shown in the table below.
+                Click on marker names in the table below to see plots of haplotype talles and thresholds in the marker detail report.
             </p>
-            <table class="half">
+            <table class="half pagination_table">
```

Comment:
Rather than printing global and marker-specific thresholds separately, just unconditionally show the table of thresholds used, after overriding globals with specifics (if any).

Also, paginate the table in case of large panels.